services/slam/builtin/orbslam_int_test.go

```diff
@@ -89,7 +89,7 @@ func testOrbslamMap(t *testing.T, svc slam.Service) {
 }
 
 // Checks the orbslam position within a defined tolerance
-func testOrbslamPosition(t *testing.T, svc slam.Service, mode, actionMode string) {
+func testOrbslamPosition(t *testing.T, svc slam.Service, mode, actionMode string, sensor string) {
```

Comment:
Where is `sensor` used?

Reply:
Ah, I was supposed to test `componentRef` against sensor. Thanks for catching this!